### PR TITLE
Style formatting

### PR DESCRIPTION
**Pull request by jonkvix**

### CODE REVIEW

File: src/components/Project.js [hunk 1]
```diff
@@ -1,45 +1,61 @@
...
     }
     let todoList = _.filter(tasks.list, t => t.state === 'to-do').map(taskDOM)
     let inProgressList = _.filter(tasks.list, t => t.state === 'in-progress').map(taskDOM)
+    let tasksComplete = _.filter(tasks.list, t => t.state === 'completed-task').map(taskDOM)
     let input
+    let desc
     return (
-      <div>
-        <h3>{name}</h3>
+      <Grid>
```
Comment (ssebelius) [inline src/components/Project.js]: I like the choice of doing this in a grid.

File: src/components/Project.js [hunk 2]
```diff
@@ -1,45 +1,61 @@
...
-            <input ref={node => { input = node }} />
-            <button type='submit'>Add Task</button>
+            <div>
+            <h5 className='task-input-font'>Task Name</h5> <br /><input type='text' size="60" placeholder='Name of the new task' ref={node => { input = node }} /><br />
+            </div>
+            <div>
+            <h5 className='task-input-font'>Task Description</h5> <br /><input type='text' size="60" placeholder="Description for the new task " ref={node => { desc = node}} />
+            </div><br />
+            <Button className='create-button' bsStyle='primary' bsSize='small' type='submit'>Create</Button>
           </form>
```
Comment (alexpeachey) [inline src/components/Project.js]: You might considering pulling this form out into it's own component.

File: src/components/Project.js [hunk 3]
```diff
@@ -1,45 +1,61 @@
...
           }}>
-            <input ref={node => { input = node }} />
-            <button type='submit'>Add Task</button>
+            <div>
+            <h5 className='task-input-font'>Task Name</h5> <br /><input type='text' size="60" placeholder='Name of the new task' ref={node => { input = node }} /><br />
+            </div>
+            <div>
+            <h5 className='task-input-font'>Task Description</h5> <br /><input type='text' size="60" placeholder="Description for the new task " ref={node => { desc = node}} />
+            </div><br />
+            <Button className='create-button' bsStyle='primary' bsSize='small' type='submit'>Create</Button>
```
Comment (ssebelius) [inline src/components/Project.js]: Might make sense to pull this all out into it's own separate component to clean it up a bit.

File: src/components/Project.js (hunk 1 above)
Comment (jonkvix) [inline src/components/Project.js]: Thanks Scott - definitely want to work with react bootstrap more in general. Pretty awesome.

File: src/components/Project.js (hunk 3 above)
Comment (jonkvix) [inline src/components/Project.js]: Totally agree with you Scott. And now as I overlook my project, I realize how much things could have been broken down further into smaller, more re-usable components (which is such a huge benefit). Thank you for the feedback.

File: src/components/Project.js (hunk 2 above)
Comment (jonkvix) [inline src/components/Project.js]: Definitely agree. And as with the comment above to Scott, I can now see quite clearly how things should have been broken down further into more components. Thanks for the feedback Alex.

**Pull request merged**